packages/catalog-process/src/services/validators.ts

```diff
@@ -202,9 +202,15 @@ export function assertHasNoDraftOrWaitingForApprovalDescriptor(
 
 export function validateRiskAnalysisSchemaOrThrow(
   riskAnalysisForm: catalogApi.EServiceRiskAnalysisSeed["riskAnalysisForm"],
-  tenantKind: TenantKind
+  tenantKind: TenantKind,
+  dateForValidation: Date
```

Comment:
See comment below. If we default to new Date, we can also remove the param here

```suggestion
  dateForExpirationValidation: Date
```